mlir/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025 Chair for Design Automation, TUM
+# All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+#
+# Licensed under the MIT License
+
+# set required cmake version
+cmake_minimum_required(VERSION 3.24...3.30)
+
+project(
+  mqt-mlir
+  LANGUAGES C CXX
+  DESCRIPTION "MQT MLIR - Interfacing between the Munich Quantum Toolkit and MLIR")
+
+set(CMAKE_CXX_STANDARD
+    17
+    CACHE STRING "C++ standard to conform to")
+
+# Add path for custom modules
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
```

Comment:
This will never add anything because there is no `cmake` directory here.
can be removed as this is no longer the top level project